client/lib/api/endpoints.dart

```diff
@@ -2,34 +2,49 @@
 
 class Endpoint {
   static const _whoMhPrefix = 'who-mh-';
+  static const _whoMh2Prefix = 'who-mh2-';
   static const _prodProjectId = 'who-mh-prod';
   static const _prodServiceUrl = 'https://covid19app.who.int';
 
+  static const _region = 'europe-west6';
```

Comment:
I'm a little wary that this parameter is repeated in the Terraform and Typescript code rather than being a shared parameter.

@rjhuijsman - is this something we can just configure through DNS? So the client connects to a custom domain and then we just point that domain to the right location? You mentioned some limitations on custom domains but I can't remember what they were.

Reply:
Great point, agreed! Per discussion on Slack, I've added a TODO here to revisit hard coding the region in the client code.